spec/serializers/framework_flag_serializer_spec.rb

```diff
@@ -29,11 +29,8 @@
     expect(result[:data][:attributes][:question_value]).to eq(framework_flag.question_value)
   end
 
-  it 'contains a `question` relationship' do
-    expect(result[:data][:relationships][:question][:data]).to eq(
-      id: framework_flag.framework_question.id,
-      type: 'framework_questions',
-    )
+  it 'contains an empty `question` relationship' do
+    expect(result[:data][:relationships][:question]).to eq({})
```

Comment:
If a relationship exists on a resource, I believe we have to return it, and never return the relationship attribute as empty, we either include it or don't include it at all. I would check if the FE requires the the question id to be on the flag, as I believe it's needed

Reply:
Ah also when querying a single move and it's flags, we no longer have access to the question as well 🤔

Reply:
Hmm.. ok, I'm not very clear on the requirements here - @smoothcontract suggested that this should be lazy-loaded (see above)?

Reply:
I think there's two options here, either have two serializers for flags one for a single Move and one for multiple where the multiple hides the question relationship, or the other approach of removing the include on the flags for questions as it isn't used anywhere and changing the relationship to: 
```
belongs_to :framework_question, serializer: FrameworkQuestionSerializer, key: :question
```

This way we always surface the question ids

Reply:
@martyn-w just an update we might not be using the question id at all on the FE from flags, so might be able to drop it all together, cc @teneightfive to confirm

Reply:
Sorry for the delay. We do need that relationship I'm afraid. It's used to build out the detailed panels on the move detail page.

#### With

<kbd>![image (1)](https://user-images.githubusercontent.com/3327997/99788685-57136e00-2b19-11eb-868c-27513232e03d.png)</kbd>

#### Without

<kbd>![image](https://user-images.githubusercontent.com/3327997/99788666-511d8d00-2b19-11eb-99fa-c14bbe09cd6e.png)</kbd>